docs/README.md

```diff
@@ -59,50 +59,45 @@ To see a list of options available:
 Read the header comment in `build.py` for detailed documentation of each option.
 
 
+
+
 ## For Developers
 
 One of our firm policies is ***EVERYTHING MUST BE DOCUMENTED***.
 
 The below are some rules to follow when updating any of the `.rst` files located in the `./docs/src/` directory tree.
 
 
-### What to Name Your `.rst` File
 
-The directory structure under the `./docs/src/` directory, and the filenames of the `.rst` files govern the eventual URLs that are generated in the HTML output.  These directories are organized so as to reflect the nature of the content.  Example:  the `.rst` files under `./docs/src/intro` contain introductory material—detailed reference material would not go there, but instead in an appropriate subdirectory of `./docs/src/details/`.  It is expected that the content and location of any new documents added would be in alignment with this directory structure, and placed and named according to their content.  Additionally, to be linked into the eventual generated documentation, the stem of the new filename needs to appear in at least one (normally *only one*) `.. toctree::` directive, normally in an `index.rst` file in the directory where it will appear in that page's table of contents (TOC).
+### reStructuredText
```

Comment:
This file contains a lot of info, but I think it's not practical.  It's just too long for most of the people to read and use. 

IMO such a guide should be very short, as few bullet points and as possible and short examples.

Reply:
Okay.